microshift_release_notes/microshift-4-19-release-notes.adoc

```diff
@@ -8,4 +8,4 @@ Do not add or edit release notes here. Edit release notes directly in the branch
 
 Release note changes should be added/edited in their own PR.
 
-This file is here to allow builds to work.
\ No newline at end of file
+This file is here to allow builds to work.
```

Comment:
Does this module get removed? It doesn't seem t be in use.

Reply:
It does not; all of the old build files stay in the repo to keep building the notes for each branch. I added a blank line at the end of that one, hence the diff.